.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/**
+.eggs/**
+dist/
```

Comment:
Why not adding `build/` as well ?

Reply:
I can add it, no problem for me. dist is the default output dir when using setup.py, I think, but I am not sure.

Reply:
I got `build/` when running `python setup.py build` (before `python setup.py install`)